src/core/prompts/tools/fetch-instructions.ts

```diff
@@ -0,0 +1,15 @@
+export function getFetchInstructionsDescription(): string {
+	return `## fetch_instructions
+Description: Request to fetch instructions to perform a task
+Parameters:
+- task: (required) The task to get instructions for.  This can take the following values:
+  create_mcp_server
+  create_mode
+  tell_a_joke
```

Comment:
Probably want to cut this and redo snapshots?

Reply:
Oops - that was left over from some early-stage testing...

Reply:
Fixed